src/test/resources/test-local.properties

```diff
@@ -0,0 +1,69 @@
+# Base Properties For This Environment
+cuc.collect.username = admin
+cuc.collect.password = secret
+
+# ActionExporter properties
+cuc.collect.actionexp.host = localhost
+cuc.collect.actionexp.port = 8141
+cuc.collect.actionexp.username = actionexporter
+cuc.collect.actionexp.password = ctp
+cuc.collect.actionexp.print.file = Documents/sftp/
+cuc.collect.actionexp.ftl.location = ./src/test/resources/actionexportersvc/
+
+# Action Service Properties
+cuc.collect.actionsvc.host = localhost
+cuc.collect.actionsvc.port = 8151
+
+# Case Service Properties
+cuc.collect.casesvc.host = localhost
+cuc.collect.casesvc.port = 8171
+
+# Collection Exercise Service Properties
+cuc.collect.collectionexercisesvc.host = localhost
+cuc.collect.collectionexercisesvc.port = 8145
+
+# Database Tool Properties
+cuc.collect.dbtool.host = localhost
+cuc.collect.dbtool.port = 9000
+
+# IAC Service properties
+cuc.collect.iacsvc.host = localhost
+cuc.collect.iacsvc.port = 8121
+cuc.collect.iacsvc.username = admin
+cuc.collect.iacsvc.password = secret
+
+# DRS Gateway properties
+cuc.collect.drsgateway.host = localhost
+cuc.collect.drsgateway.username = gateway
+cuc.collect.drsgateway.password = ctp
+
+# Sample Service properties
+cuc.collect.samplesvc.host = localhost
+cuc.collect.samplesvc.port = 8125
+cuc.collect.samplesvc.xsd.location = samplesvc/xsd/inbound/
+cuc.collect.samplesvc.xml.location = ./src/test/resources/samplesvc/
+cuc.collect.samplesvc.sftp.survey = ./Documents/sftp/Sample-Files/%s/
+cuc.collect.samplesvc.csv.location = src/test/resources/samplesvc/
+
+# SDX Gateway properties
+cuc.collect.sdxgateway.port = 8191
+cuc.collect.sdxgateway.host = localhost
+cuc.collect.sdxgateway.username = gateway
+cuc.collect.sdxgateway.password = ctp
+
+# SFTP properties
+cuc.sftp.server = localhost
+cuc.sftp.port = 122
+cuc.sftp.samplesvc.username = centos
+cuc.sftp.samplesvc.password = JLibV2&XD,
+cuc.sftp.actionexp.username = centos
+cuc.sftp.actionexp.password = JLibV2&XD,
+
```

Comment:
I assume we're ok with these passwords being public?

Reply:
It's here https://github.com/ONSdigital/rm-docker-dev/blob/master/docker-compose-dev_env.yml so probably ok